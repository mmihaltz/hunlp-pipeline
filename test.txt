Egy kisbolygó vagy aszteroida a törpebolygónál kisebb, szabálytalan alakú, szilárd anyagú égitest, mely csillag körül kering. A legtöbb kisbolygó feltehetően a protoplanetáris korongból származik, melyek nem álltak össze bolygóvá a csillagrendszer kialakulásakor. Néhányuk saját holddal is rendelkezik.

A mi Naprendszerünkben a kisbolygók többsége a kisbolygóövben található; ellipszis alakú pályán keringenek a Nap körül, a Mars és a Jupiter pályája között illetve a Kuiper-övben azaz a Pluto környékén helyezkednek el.
