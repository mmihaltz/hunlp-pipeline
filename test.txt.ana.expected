Egy	egy	ART	egy/ART
kisbolygó	kisbolygó	NOUN	kisbolygó/ADJ
vagy	vagy	CONJ	vagy/CONJ
aszteroida	aszteroida	NOUN	aszteroida/NOUN
a	a	ART	a/ART
törpebolygónál	törpebolygónál	ADJ<CAS<ADE>>	0
kisebb	kis	ADJ	kis/ADJ[COMPAR]/ADJ
,	,	PUNCT	,/PUNCT
szabálytalan	szabálytalan	ADJ	szabálytalan/ADJ
alakú	alakú	ADJ	alakú/ADJ
,	,	PUNCT	,/PUNCT
szilárd	szilárd	ADJ	szilárd/ADJ
anyagú	anyagú	ADJ	anyagú/ADJ
égitest	égitest	NOUN<CAS<ACC>>	égitest/NOUN
,	,	PUNCT	,/PUNCT
mely	mely	NOUN	mely/NOUN
csillag	csillag	NOUN	csillag/NOUN
körül	körül	POSTP	körül/POSTP
kering	kering	VERB	kering/VERB
.	.	PUNCT	./PUNCT

A	a	ART	a/ART
legtöbb	sok	NUM	sok/NUM[SUPERLAT]/NUM
kisbolygó	kisbolygó	NOUN	kisbolygó/ADJ
feltehetően	feltehető	ADV	feltehető/ADJ[MANNER]/ADV
a	a	ART	a/ART
protoplanetáris	protoplanetáris	ADJ	0
korongból	korong	NOUN<CAS<ELA>>	korong/NOUN<CAS<ELA>>
származik	származik	VERB	származik/VERB
,	,	PUNCT	,/PUNCT
melyek	mely	NOUN<PLUR>	mely/NOUN<PLUR>
nem	nem	ADV	nem/ADV
álltak	áll	VERB<PAST><PLUR>	áll/VERB<PAST><PLUR>
össze	össze	PREV	össze/PREV
bolygóvá	bolygó	VERB<COND><DEF>	bolygó/ADJ<CAS<TRA>>
a	a	ART	a/ART
csillagrendszer	csillagrendszer	NOUN	csillagrendszer/NOUN
kialakulásakor	kialakulás	NOUN<POSS><CAS<TEM>>	kialakulás/NOUN<POSS><CAS<TEM>>
.	.	PUNCT	./PUNCT

Néhányuk	néhány	NOUN<POSS<PLUR>>	néhány/NOUN<POSS<PLUR>>
saját	saját	NOUN	saját/NOUN
holddal	hold	NOUN<CAS<INS>>	hold/NOUN<CAS<INS>>
is	is	ADV	is/ADV
rendelkezik	rendelkezik	VERB	rendelkezik/VERB
.	.	PUNCT	./PUNCT

A	a	ART	a/ART
mi	mi	NOUN<PERS<1>><PLUR>	mi/NOUN<PERS<1>><PLUR>
Naprendszerünkben	naprendszer	NOUN<POSS<1><PLUR>><CAS<INE>>	naprendszer/NOUN<POSS<1><PLUR>><CAS<INE>>
a	a	ART	a/ART
kisbolygók	kisbolygó	NOUN<PLUR>	kisbolygó/ADJ<PLUR>
többsége	többség	NOUN<POSS>	többség/NOUN<POSS>
a	a	ART	a/ART
kisbolygóövben	kisbolygóövben	NOUN<CAS<INE>>	0
található	található	ADJ	található/ADJ
;	;	PUNCT	;/PUNCT
ellipszis	ellipszis	NOUN	ellipszis/NOUN
alakú	alakú	ADJ	alakú/ADJ
pályán	pálya	NOUN<CAS<SUE>>	pálya/NOUN<CAS<SUE>>
keringenek	kering	VERB<PLUR>	kering/VERB<PLUR>
a	a	ART	a/ART
Nap	Nap	NOUN	Nap/NOUN
körül	körül	POSTP	körül/POSTP
,	,	PUNCT	,/PUNCT
a	a	ART	a/ART
Mars	Mars	NOUN	Mars/NOUN
és	és	CONJ	és/CONJ
a	a	ART	a/ART
Jupiter	Jupiter	NOUN	Jupiter/NOUN
pályája	pálya	NOUN<POSS>	pálya/NOUN<POSS>
között	között	POSTP	között/POSTP
illetve	illetve	CONJ	illetve/CONJ
a	a	ART	a/ART
Kuiper-övben	Kuiper-övben	NOUN<CAS<INE>>	0
azaz	azaz	CONJ	azaz/CONJ
a	a	ART	a/ART
Pluto	Pluto	NOUN	0
környékén	környék	NOUN<POSS><CAS<SUE>>	környék/NOUN<POSS><CAS<SUE>>
helyezkednek	helyezkedik	VERB<PLUR>	helyezkedik/VERB<PLUR>
el	el	PREV	el/PREV
.	.	PUNCT	./PUNCT

